src/test/regress/expected/isolation_enable_ddl_propagation.out

```diff
@@ -0,0 +1,16 @@
+Parsed test spec with 1 sessions
+
+starting permutation: s1-begin s1-commit
+result
+---------------------------------------------------------------------
+t
+t
+t
+(3 rows)
+
+step s1-begin:
+ BEGIN;
+
+step s1-commit:
+ COMMIT;
+
```

Comment:
We can reduce the two test outputs into one by changing the queries to the following:

```sql
SELECT bool_and(result='ALTER SYSTEM') AS all_ok FROM  run_command_on_all_nodes('ALTER SYSTEM SET citus.enable_ddl_propagation TO ON');

SELECT bool_and(result='t') AS all_ok FROM run_command_on_all_nodes('SELECT pg_reload_conf()');
```